contracts/modules/assessment/IndividualClaims.sol

```diff
@@ -259,13 +264,13 @@ contract IndividualClaims is IIndividualClaims, MasterAwareV2 {
       lastClaimSubmissionOnCover[coverId] = ClaimSubmission(uint80(claims.length), true);
     }
 
-    ICover coverContract = cover();
+    ICoverProducts coverProductsContract = coverProducts();
     CoverData memory coverData = cover().coverData(coverId);
     CoverSegment memory segment = cover().coverSegmentWithRemainingAmount(coverId, segmentId);
 
     {
-      Product memory product = coverContract.products(coverData.productId);
-      ProductType memory productType = coverContract.productTypes(product.productType);
+      Product memory product = coverProductsContract.products(coverData.productId);
+      ProductType memory productType = coverProductsContract.productTypes(product.productType);
```

Comment:
We seem to have a pattern of fetching the product and the type (saw this in some other contract). Consider creating a function `getProductWithType` to fetch both in a single call.

Reply:
applied